articles/getting-started/how-to-get-help.md

```diff
@@ -15,15 +15,15 @@ Find out whether the service is up, down, sideways, on vacation or going slow by
 |:------------------------------------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------------------|
 | **Upgrades, Downgrades (standard plans)**       | See the **[Billing](https://app.sparkpost.com/account/billing)** page under Account in the SparkPost UI                                                          |
 | **Upgrades - Large Plans & Add-ons**            | **[Contact Sales](mailto:insidesales@sparkpost.com?subject=Account%20Change)**                                                                                   |
-| **Cancel your Service**                         | **[Submit a support request](https://www.sparkpost.com/submit-a-ticket?email[subject]=Web:%20Cancel%20Service%20Request:%20Selfserve%20Account)** |
-| **Cancel your Service - Large Plans >2M/month** | **[Submit a support request](https://www.sparkpost.com/submit-a-ticket?email[subject]=Web:%20Cancel%20Service%20Request:%20Large%20Account)**     |
-| **Billing Inquiries**                           | **[Contact our Billing Team](mailto:billing@sparkpost.com?subject=Billing%20Request)**           |
+| **Cancel your Service**                         | **[Submit a support request](https://app.sparkpost.com/dashboard/?supportTicket=true&supportIssue=account_cancellation)** |
+| **Cancel your Service - Large Plans >2M/month** | **[Submit a support request](https://app.sparkpost.com/dashboard/?supportTicket=true&supportIssue=account_cancellation)**     |
```

Comment:
Do we still need 2 entries for "cancel your service" ? They both go to the same support form now so...

Reply:
we dont need the 2 now - we used to run a different process on the large ones